### DISCUSSION TITLE
Missing last step 2024 Silverado

### DISCUSSION THREAD

**Discussion by rrmccabe** [agronick/aa-torque#147] [Q&A]
Trying to get Torque setup on my new truck.  Samsung S21 phone. Was able to install the APK and that seems to be running fine. I believe I got the permissions correct during this process as well.  I have tried connecting the normal wireless way and also USB.

The documentation says "You can find the AA Torque in the menu on the lower right in Android Auto, the one with the dashboard clock on it."

This is the where I am now following. I do not see AA torque anywhere. 

Thanks,

Rich

**Comment by rrmccabe** [comment]
Additional info after scouring through some other threads.  It is not showing up in launcher even when connected.

I did do force update and install but no matter how many times i do that it shows update available. Am I not getting the latest version installed?

**Comment by agronick** [comment]
It is a hack. You need to bypass certain restrictions Google puts in place. It is much harder on newer phones. AAWireless is the easiest way to get it working.
